docs/general/foundation/testnet/upgrade-log.md

```diff
@@ -302,8 +302,8 @@ For a more detailed explanation of the parameters below, see the [Genesis Docume
     "scale": "1229"
   }
   ```
-  For example, this element specifies that the staking reward is 12.29% until
-  epoch `14226`.
+  For example, this element specifies that the staking reward is 0.001229% per
+  epoch until epoch `14226`.
```

Comment:
Pretty sure this was wrong

Reply:
Yes, this was wrong. Thanks for noticing and fixing this!